src/reducers/SnackbarStore.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
```

Comment:
From file structure point of view, why do we keep stores like `MyStore.tsx` in `/reducers` folder? Shouldn't be `/stores` folder?  Looking at the`SnackbarStore` code, it doesn't provide _reducer like_ functionalities, right? I'm just curious if this is common practice.

Reply:
That's right, going to change it 👍